packages/kas/src/parser.js

```diff
@@ -1,15 +1,5 @@
 // This is a @generated file
-var _, KAS = {};
-
-if (typeof module === "object" && module.exports) {
-    _ = require("underscore");
-    module.exports = KAS;
-} else {
-    _ = window._;
-    window.KAS = KAS;
-}
-
-(function(KAS) {
+import _ from "underscore";
```

Comment:
Did you get these changes by re-generating this file?  If not, we probably should...

Reply:
I did not, but I did update parser-generator.js in the next PR to generate these files without any changes.